.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Set up ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 3.0.4
+          ruby-version: 3.2.1
```

Comment:
Aynı işlemi rspec tarafı içinde yapmalıyız. Ayrıca rubocop hatasını gidermek için de rubocop -A çalıştırırsak iki testten de geçeceği kanısındayım